lib/logstash/outputs/logstash.rb

```diff
@@ -83,10 +91,15 @@ def close
     logger.debug('inner HTTP output plugin has been closed')
   end
 
+  DEFAULT_PORT = 9800.freeze
+
   def inner_http_output_options
+    urls = validate_and_normalize_hosts
+    logger.warn("Currently, single host is supported. Using #{urls.first.inspect} and ignoring rest of them.") if urls.size > 1
```

Comment:
review note: instead of raising a config error, preferred to use first element and warn. This will be removed with HA support work

Reply:
If only a single host is supported now, then we should simply not add the `:list` property to the config option. I acknowledge that conflicts with the plural nature of `hosts` config option being introduced here, and assert that we should not introduce a plural `hosts` until we support more than one host.

Reply:
IMHO, I don't see the harm of adding a list type option here when we are going to be implementing the support for multiple hosts immediately after, in a follow up release. So, as long as not using `:list` won't require the user to manually have to make changes to the setting once they upgrade to a version that uses`:list`, then I'm not making it so for now.

Reply:
Removed `:list` on hosts based on the comments here and will add it when we support multi-host.